apps/web/components/eventtype/EventSetupTab.tsx

```diff
@@ -139,7 +139,10 @@ export const EventSetupTab = (
       return true;
     });
 
-    const defaultValue = locationOptions.find((item) => item.label === "video")?.options;
+    const defaultValue = locationOptions
+      .find((item) => item.label === "video")
```

Comment:
I am not in favour of identifying a video app from a label. There should be a better way to do it.
Looks good otherwise

Reply:
Agreed with Hariom, you can safely use category/categories for this. Otherwise, great job!